easybuild/tools/github.py

```diff
@@ -1221,14 +1223,19 @@ def not_eligible(msg):
 
     # check github mergeable state
     msg_tmpl = "* mergeable state is clean: %s"
+    mergeable = False
     if pr_data['merged']:
         print_msg(msg_tmpl % "PR is already merged", prefix=False)
     elif pr_data['mergeable_state'] == GITHUB_MERGEABLE_STATE_CLEAN:
         print_msg(msg_tmpl % "OK", prefix=False)
+        mergeable = True
     else:
         reason = "FAILED (mergeable state is '%s')" % pr_data['mergeable_state']
         res = not_eligible(msg_tmpl % reason)
 
+    if failed_status_last_commit and mergeable:
+        print_msg("\nThis PR is mergeable but the test suite has a failed status. Try syncing the PR.", prefix=False)
```

Comment:
Maybe `Try syncing the PR` should be a bit more specific, it could say "`Try syncing the PR with the develop branch using 'eb --sync-pr-with-develop XXX'`"?

Reply:
20fdb74